### ISSUE TITLE
When silence is eerie

### ISSUE THREAD

**Issue by cinnamoner** [cinnamoner/Clipboard#4]

**Comment by cinnamoner** [comment]
Talk! I'm listening. 

@feren @Elbullazul Speak your mind. Here.

**Comment by cinnamoner** [comment]
> KDE has it, XFCE (!!) has it being a super lightweight environment. Windows 98 had it and every release since, OSX has had it since the mid-90's. I can't get it into my head why any modern desktop environment (cough GNOME3, LXDE, Cinnamon, MATE, ……) wouldn't implement this.

**Comment by cinnamoner** [comment]
Any DE for GNU with Linux that:

1. Doesn't have a proper destop grid.
2. Aero incapable.

is not good enough.

Aero is not a set of desktop features. According to Intel it's a theme. A theme defined by 4 characteristics:
1. Transparent window decoration.
2. Ability to control parameters of 1 (blur, color ..
4. Window fx.
5. Ability to control the dimensions of close, minimize, maximize independently from one another.

**Comment by cinnamoner** [comment]
Are you aware that GNU can work with other kernels than Linux?

**Comment by cinnamoner** [comment]
Aero and the mechanics of Windows are the best ever created for the Desktop. I doubt that mobile and desktop convergence is good.

**Comment by cinnamoner** [comment]
An idea just crossed your mind? Post it here.

**Comment by Elbullazul** [comment]
@cinnamoner bsd and 4L anmong them

I'd rather have a stable DE than a pretty one (flash and alpha) but buggy. that's where development happens

**Comment by cinnamoner** [comment]
@Elbullazul So you have read it. I agree. What distro do you recommend then. It has to be aero capable though. Thanks.

**Comment by cinnamoner** [comment]
The latest iterations of Neon are not entirely bug-free but nothing critical. Mint themselves have bugs of their own. What's your favourite distro anyways?
***
Aero and desktop grid are important to me.

**Comment by cinnamoner** [comment]
Even corporate OSs like ... have some bugs.

**Comment by cinnamoner** [comment]
Since I've brought up Neon, there's no way to form an opinion, unless one tires the latest. The progress has been and is astonishing.

**Comment by cinnamoner** [comment]
@Elbullazul An MX 16 KDE 4 is readily available. Not officially supported though. Wonder whether it's worth downloading. @feren Huh? 
***
@Elbullazul Does Xfce support Emerald themes?

**Comment by cinnamoner** [comment]
@feren You installed Thunar in Cinnamon and I (accidentally) installed Nemo in Neon. Wonder whether Stak can somehow be installed in Plasma or Xfce.

**Comment by cinnamoner** [comment]
Ideas ... as they come ... no particular order

**Comment by cinnamoner** [comment]
4L. What's that?

**Comment by Elbullazul** [comment]
@cinnamoner about xfce I have no idea about it...

https://en.wikipedia.org/wiki/L4_microkernel_family

**Comment by cinnamoner** [comment]
Yeah, I had run a search a few minutes after reading your post. It's a microkernel alright for ARM. I'm more intersted in kernels for the desktop though. But thanks.

Don't know what happened with Steve's channel. It went down while I was watching a video of his.

**Comment by cinnamoner** [comment]
Misspelled a lot on this thread. Ha.

**Comment by cinnamoner** [comment]
@Elbullazul What is _SquashFS_? This one might actually lead somewhere.

**Comment by Elbullazul** [comment]
@cinnamoner it's a file system structure that the kernel decompresses into memory and that contains the 'live environment' of live distros (may have other usages)

**Comment by cinnamoner** [comment]
@Elbullazul Then have a look here. Do you think it's actually usable. Consult associated links.
https://github.com/AdrianTM/remastering-script

**Comment by cinnamoner** [comment]
Wonder whether it has similar functionality to Constructor.

**Comment by cinnamoner** [comment]
Say I decompress a distro into the RAM then I remaster it. Can I get an .iso?

**Comment by cinnamoner** [comment]
http://www.mepis.org/docs/en/index.php?title=Remastering_script

**Comment by cinnamoner** [comment]
So?

**Comment by cinnamoner** [comment]
@bluedxca93 Check out this thread, please. And tell me what you think.

**Comment by cinnamoner** [comment]
@feren You're also welcome to opinionate. After all, it's your kind of business, I guess. And yes I like Sudo although he rambles a lot. But he's a neat guy, I guess.

**Comment by cinnamoner** [comment]
Yeah, I'm constantly parsing the place. https://github.com/MX-Linux

**Comment by cinnamoner** [comment]
These guys are using Vivaldi. It's a good browser. https://github.com/KaOS-Community-Packages/vivaldi-snapshot

**Comment by cinnamoner** [comment]
@Elbullazul Interesting package. Hahaha! https://github.com/KaOS-Community-Packages/icoutils

**Comment by cinnamoner** [comment]
@Elbullazul Seems you've left Aple and you're working for Google now.

**Comment by feren** [comment]
@cinnamoner Literally every distribution with a Filesystem uses SquashFS...

Also, https://sourceforge.net/projects/feren-os-development-builds/files/ (Pre-BETA is out!)

**Comment by cinnamoner** [comment]
@feren Hey, that's great news about your OS.

**Comment by cinnamoner** [comment]
@Elbullazul Are you skilled in Qtcurve? Moony has a 7 in the store. Interested? Would you check it? At any rate, only do what you like.

**Comment by bluedxca93** [comment]
@cinnamoner qtcurve yeah its possible. you start from a light grey.

**Comment by feren** [comment]
feren OS 2017.0 BETA is out

**Comment by cinnamoner** [comment]
@bluedxca93 What engine powers qtcurve? Aurorae is sluggish. Smargd, which I currently use for emerald themes on Neon, is snappy n fast.

**Comment by cinnamoner** [comment]
@feren Glad about the beta. Just take it slowly. No bugs should make it into the final release.

**Comment by cinnamoner** [comment]
@feren So, what's Sudo's distro of choice? Peppermint. MX. Manjaro Xfce. What do you think about MX? It's themable and re-spinable. Kinda like Xfce because it's trouble-free and light. When it's bearbones it looks crappy but one can theme it beyond the wildest. I think it can take emerald themes. And the menu is drag-resizable (rings a bell).

**Comment by feren** [comment]
@cinnamoner Manjaro
MX: Weird Panel Configuration...

**Comment by cinnamoner** [comment]
@feren Have you noticed anything weird when signing in to GitHub on mobile? Cause it says my browser is no longer supported. Which I doubt. It's Chrome on Android and I see no reason why it wouldn't be supported. What's even more strange is that it says to update to the latest Chrome or Firefox and there's an old Internet Explorer icon with an exclamation mark, as if I would have been signed in from such a deprecated browser. I've got no Internet Explorer on Android. Just how on earth could I have it on Android?
What's your theory? I've purged the device but to no avail. Git lets me sign in eventually but with some hassle.

**Comment by cinnamoner** [comment]
Oh, I've got a header all the time now saying my browser is not supported.

**Comment by cinnamoner** [comment]
@Elbullazul GitHub says I'm on a deprecated Internet Explorer. I'm not on such a thing and I'm on Android. Strange.

**Comment by cinnamoner** [comment]
And the header gives two links: one leading to Firefox's and the other to Chrome's. The links seem genuine but I don't trust my own mother although I've got no reason not to. Hahaha!

**Comment by cinnamoner** [comment]
The panel in MX can be configured. The distro is unbelievable. Manjaro. Some people.

**Comment by cinnamoner** [comment]
Definitely strange. Cannot switch the dashboard and cannot sign out. If it's a hacker than he does a poor job. Hahaha.

**Comment by cinnamoner** [comment]
The deal is I use the internet front-end with Google's engine. Not Chrome as such. Is that the cause or what? The internet front-end manages everything and never felt the need to use Chrome as a standalone. Understand? It's Android though. All the way. Nobody at home today, I guess.

**Comment by cinnamoner** [comment]
Feren, I need a good mobile OS, preferably not proprietary crappware. Open source and free (as in Stallman free).

**Comment by cinnamoner** [comment]
It's all about the money, damn it. And when you finally have enough of it, if ever, you're time is up or your health is gone or I'm signing out. See you.

**Comment by cinnamoner** [comment]
Can't edit posts anymore. Can delete them. Probably can even push a repo though. It's ridiculous and absurd. Anyways, I had enough. I'm upset alright.

**Comment by cinnamoner** [comment]
No emoji functionality anymore either. Good. And I'm not going to open the browser cause I don't feel like it. Alright?

**Comment by Elbullazul** [comment]
@cinnamoner free mobile OSes are almost inexistant. The ones I know are Firefox OS (deprecated) and Sailfish OS (not really shure)

**Comment by cinnamoner** [comment]
Firefox OS was bought out.

**Comment by cinnamoner** [comment]
Maybe that plasma mobile thing. For what it packs, I'm amazed how stable Plasma has become. It's a very complex desktop, hence it's prone to be be buggy. It's like a clock. That many pieces it has and all have to work in harmony. It's capable but probably overly complex.

**Comment by cinnamoner** [comment]
It does the job. Doesn't crashes. Not yet perfect. Awesome though. One can checksum and compare checksums from any file's properties. SHA256 included.

**Comment by cinnamoner** [comment]
@Elbullazul Sorry, I need to rest my brains. See you old friend.

**Comment by cinnamoner** [comment]
I'm on Android. However, Git says I'm on a deprecated Internet Explorer. Some functionality is lost. Any theory? I'm not using IE in Windows itself. I'm utterly puzzled.

**Comment by feren** [comment]
@cinnamoner What Android ve

**Comment by cinnamoner** [comment]
If I instal Firefox or Chrome, there's no problem though. Kit.

**Comment by cinnamoner** [comment]
@feren Fun fact: all the mobile devices I've got were presents. I'm not wasting money on tracking devices.
***
Do you pay for Facebook's services? No. Then how do they make money, which they obviously need to exist and get even better in whatever they are doing. Where does the money come form? No need for an answer. Just reflect. Facebook is for the herd. I'm quite done on the subject. I've got nothing to hide, hence I'm going out naked to do some shopping. Get it?

**Comment by cinnamoner** [comment]
You don't need their software installed for them to be able to track you. They're everywhere. I've detected active tracking from them even on Open Desktop. I'm done. Use specialized softer and monitor the traffic. You'll be surprised.

**Comment by cinnamoner** [comment]
You don't need their software installed for them to be able to track you. They're everywhere. I've detected active tracking from them even on Open Desktop. I'm done. Use specialized software and monitor the traffic. You'll be surprised. If you want your kids with cyber-implants, that's entirely your choice, cause we're heading right there.

**Comment by cinnamoner** [comment]
Right there https://youtu.be/WZEJ4OJTgg8

**Comment by cinnamoner** [comment]
Something like that is not as farfetched as you may think.

**Comment by cinnamoner** [comment]
If you don't fight back NOW, then resistance will be futile. Extract the paradigms from my posts. Don't take them letter by letter. It's for the last time I'm bringing it up.

**Comment by cinnamoner** [comment]
Oh, one last thing: data mining and aggregation of it, irrelevant of purpose, is bad practice and potentially dangerous. History has demonstrated it.

>Our privacy is more important than our fear that something bad might happen.
